pkgs/applications/office/libreoffice/default-gen-shell.nix

```diff
@@ -19,9 +19,11 @@ stdenv.mkDerivation {
     builder = ./download-list-builder.sh;
   };
 
+  buildInputs = [ python36 ];
+
   shellHook = ''
     function generate {
-      ./generate-libreoffice-srcs.sh | tee libreoffice-srcs.nix
+      python3 generate-libreoffice-srcs.py > libreoffice-srcs.nix
```

Comment:
Why not add `#! /usr/bin/env python3` in the script, too? I think most NixPkgs developers have `/usr/bin/env` (it is default-on in NixOS)